gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/transports/rest.py.j2

```diff
@@ -92,7 +94,7 @@ class {{ service.name }}RestInterceptor:
         return request, metadata
 
     {% if not method.void %}
-    def post_{{ method.name|snake_case }}(self, response: {{method.output.ident}}) -> {{method.output.ident}}:
+    def post_{{ method.name|snake_case }}(self, response: Union[{{method.output.ident}}, rest_streaming.ResponseIterator]) -> Union[{{method.output.ident}}, rest_streaming.ResponseIterator]:
```

Comment:
Strictly speaking, this should be a `rest_streaming.ResponseIterator` only if this is a server streaming method.

Reply:
Done